treeherder/seta/analyze_failures.py

```diff
@@ -59,8 +59,16 @@ def get_failures_fixed_by_commit():
     failures = {}
     # We're assuming that sheriffs always anotate failed jobs correctly using "fixed by commit"
     for job_note in models.JobNote.objects.filter(failure_classification=2):
```

Comment:
How about we try to fix Bug 1338570 now also? I *believe* you'd just add:

for job_note in models.JobNote.objects.filter(failure_classification=2).select_related('job'):